src/utils/components/ExternalLink/ExternalLink.tsx

```diff
@@ -1,5 +1,6 @@
 import * as React from 'react';
```

Comment:
can you please get rid of the * here?

Reply:
Yep, will do it. Thanks!
Just a note, maybe you've already realized, too, that we have sooo much stuff like this across whole codebase and I am not sure why we haven't imported only what we need from the beginning. Our whole codebase deserves refactoring, and not only regarding this, but we also have so much repeating code everywhere :D

Reply:
we had an issue with typescript at the beginning